components/grid/templates/overview.md

```diff
@@ -29,6 +29,8 @@ The Grid component can use templates for:
 
 * [filter]({%slug grid-templates-filter%})  - the content of the filter cell or filter menu where you can implement custom rendering and logic for the filters.
 
+* [no data available]({%slug grid-templates-no-data%}) - while the data is still loading or when you try to filter on a non-existing item, you can write any message.
```

Comment:
Please clarify this sentence and add to it (maybe as the very first thing) something like "The grid Data collection is empty or null" to clearly denote the main use case.

Reply:
Done